GitUI/CommandsDialogs/FormCleanupRepository.cs

```diff
@@ -78,17 +90,32 @@ private CleanMode GetCleanMode()
             throw new NotSupportedException($"Unknown value for {nameof(CleanMode)}.");
         }
 
-        private string? GetPathArgumentFromGui()
+        private string? GetInclusivePathArgumentFromGui()
         {
-            if (!checkBoxPathFilter.Checked)
+            if (!checkBoxIncludePathFilter.Checked)
             {
                 return null;
             }
 
             // 1. get all lines from text box which are not empty
             // 2. wrap lines with ""
             // 3. join together with space as separator
-            return string.Join(" ", textBoxPaths.Lines.Where(a => !string.IsNullOrEmpty(a)).Select(a => string.Format("\"{0}\"", a)));
+            return string.Join(" ", textBoxIncludePaths.Lines.Where(a => !string.IsNullOrEmpty(a)).Select(a => string.Format("\"{0}\"", a)));
+        }
+
+        private string? GetExclusivePathArgumentFromGui()
+        {
+            if (!checkBoxExcludePathFilter.Checked)
+            {
+                return null;
+            }
+
+            // 1. get all lines from text box which are not empty
+            // 2. Prepend lines with '-e /'
```

Comment:
Could you please clarify why we're prepending `/`? This means root of the drive, not the root of the repo.

Reply:
Had some trouble getting the patterns to work, the `/` is definitely not necessary on further testing.